server/api.go

```diff
@@ -507,11 +507,13 @@ func (p *Plugin) handleConfig(w http.ResponseWriter) error {
 
 	type config struct {
 		clientConfig
-		SkuShortName string `json:"sku_short_name"`
+		SkuShortName        string `json:"sku_short_name"`
+		HostControlsAllowed bool
 	}
 	ret := config{
-		clientConfig: p.getConfiguration().getClientConfig(),
-		SkuShortName: skuShortName,
+		clientConfig:        p.getConfiguration().getClientConfig(),
+		SkuShortName:        skuShortName,
+		HostControlsAllowed: p.licenseChecker.HostControlsAllowed(),
```

Comment:
Maybe we could consider moving all of this into `getClientConfig` instead of having yet another wrapping object.

Reply:
@streamer45 Let me know what you think of doing it this way

Reply:
Ah forgot we needed the API to properly calculate it (hence the wrapper). But yeah, making it a plugin method works fine, thanks!